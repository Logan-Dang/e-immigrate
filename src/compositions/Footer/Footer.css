.footer {
    height: 132px;
    background-color: var(--color-brightSky);
    margin: 0 -2rem;
    padding: 18px 2rem;
    display: grid;
    grid-template-rows: 50% 35% 15%;
    grid-template-rows: auto;
    align-items: center;
    justify-content: center;
}

.footer .imgRow {
    display: grid;
    grid-template-columns: auto auto;
    gap: 18px;
    align-items: center;
}

.footer .imgRow .ciitLogo {
    width: 48px;
    height: auto;
    justify-self: end;
}

.footer .imgRow .ceticpLogo {
    width: 48px;
    height: auto;
    justify-self: start;
}

.footer .textRow1 {
    display: grid;
    grid-template-rows: auto;
    justify-content: center;
    text-align: center;
}

.footer .textRow2 {
    text-align: center;
}
