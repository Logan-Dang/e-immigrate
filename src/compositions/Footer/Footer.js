import React from 'react';
import ciitLogo from '../../data/images/CiiT Logo.png';
import ceticpLogo from '../../data/images/CET-ICP Logo.png';
import './Footer.css';

const Footer = () => {
    return (
        <div className="footer">
            <div className="imgRow">
                <img className="ciitLogo" src={ciitLogo} alt="CiiT Logo" />
                <img
                    className="ceticpLogo"
                    src={ceticpLogo}
                    alt="CET-ICP Logo"
                />
            </div>
            <div className="textRow1">
                <div>
                    CIIT was designed by the Center for Employment Training -
                </div>
                <div>Immigration & Citizenship Program (CET-ICP)</div>
            </div>
            <div className="textRow2">
                CET-ICP is a 501C3 Nonprofit Organization
            </div>
        </div>
    );
};

export default Footer;
