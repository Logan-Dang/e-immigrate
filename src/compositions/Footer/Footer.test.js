import React from 'react';
import Footer from './Footer';

import renderer from 'react-test-renderer';

it('App renders correctly', () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
});
