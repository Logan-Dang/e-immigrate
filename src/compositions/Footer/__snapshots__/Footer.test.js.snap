// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<div
  className="footer"
>
  <div
    className="imgRow"
  >
    <img
      alt="CiiT Logo"
      className="ciitLogo"
      src="CiiT Logo.png"
    />
    <img
      alt="CET-ICP Logo"
      className="ceticpLogo"
      src="CET-ICP Logo.png"
    />
  </div>
  <div
    className="textRow1"
  >
    <div>
      CIIT was designed by the Center for Employment Training -
    </div>
    <div>
      Immigration & Citizenship Program (CET-ICP)
    </div>
  </div>
  <div
    className="textRow2"
  >
    CET-ICP is a 501C3 Nonprofit Organization
  </div>
</div>
`;
